CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP0` AS SELECT A.`seq_region_id` AS `seq_region_id_1021`,A.`status` AS `status_1021`,A.`canonical_annotation` AS `canonical_annotation_1021`,A.`display_xref_id` AS `display_xref_id_1021`,A.`gene_id` AS `gene_id_1021`,A.`seq_region_end` AS `seq_region_end_1021`,A.`source` AS `source_1021`,A.`seq_region_start` AS `seq_region_start_1021`,A.`description` AS `description_1021`,A.`analysis_id` AS `analysis_id_1021`,A.`biotype` AS `biotype_1021`,A.`canonical_transcript_id` AS `canonical_transcript_id_1021`,A.`is_current` AS `is_current_1021`,A.`seq_region_strand` AS `seq_region_strand_1021` FROM `homo_sapiens_core_60_37e`.`gene` AS A;
CREATE INDEX I_0 ON `homo_sapiens_core_60_37e`.`TEMP0`(`analysis_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP1` AS SELECT A.*,B.`db` AS `db_102`,B.`db_version` AS `db_version_102`,B.`module` AS `module_102`,B.`gff_source` AS `gff_source_102`,B.`module_version` AS `module_version_102`,B.`logic_name` AS `logic_name_102`,B.`gff_feature` AS `gff_feature_102`,B.`program_version` AS `program_version_102`,B.`db_file` AS `db_file_102`,B.`program_file` AS `program_file_102`,B.`created` AS `created_102`,B.`program` AS `program_102`,B.`parameters` AS `parameters_102` FROM `homo_sapiens_core_60_37e`.`TEMP0` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`analysis` AS B ON A.`analysis_id_1021`=B.`analysis_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP0`;
CREATE INDEX I_1 ON `homo_sapiens_core_60_37e`.`TEMP1`(`gene_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP2` AS SELECT A.*,B.`created_date` AS `created_date_1024`,B.`stable_id` AS `stable_id_1024`,B.`modified_date` AS `modified_date_1024`,B.`version` AS `version_1024` FROM `homo_sapiens_core_60_37e`.`TEMP1` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`gene_stable_id` AS B ON A.`gene_id_1021`=B.`gene_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP1`;
CREATE INDEX I_2 ON `homo_sapiens_core_60_37e`.`TEMP2`(`seq_region_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP3` AS SELECT A.*,B.`name` AS `name_1053`,B.`length` AS `length_1053`,B.`coord_system_id` AS `coord_system_id_1053` FROM `homo_sapiens_core_60_37e`.`TEMP2` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`seq_region` AS B ON A.`seq_region_id_1021`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP2`;
CREATE INDEX I_3 ON `homo_sapiens_core_60_37e`.`TEMP3`(`coord_system_id_1053`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP4` AS SELECT A.*,B.`rank` AS `rank_107`,B.`name` AS `name_107`,B.`attrib` AS `attrib_107`,B.`species_id` AS `species_id_107`,B.`version` AS `version_107` FROM `homo_sapiens_core_60_37e`.`TEMP3` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`coord_system` AS B ON A.`coord_system_id_1053`=B.`coord_system_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP3`;
CREATE INDEX I_4 ON `homo_sapiens_core_60_37e`.`TEMP4`(`seq_region_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP5` AS SELECT A.*,B.`sequence` AS `sequence_1013` FROM `homo_sapiens_core_60_37e`.`TEMP4` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`dna` AS B ON A.`seq_region_id_1021`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP4`;
CREATE INDEX I_5 ON `homo_sapiens_core_60_37e`.`TEMP5`(`seq_region_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP6` AS SELECT A.*,B.`sequence` AS `sequence_1015`,B.`n_line` AS `n_line_1015` FROM `homo_sapiens_core_60_37e`.`TEMP5` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`dnac` AS B ON A.`seq_region_id_1021`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP5`;
RENAME TABLE `homo_sapiens_core_60_37e`.`TEMP6` TO `homo_sapiens_core_60_37e`.`gene`;
CREATE INDEX I_6 ON `homo_sapiens_core_60_37e`.`gene`(`gene_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP7` AS SELECT A.`gene_id_1021` FROM `homo_sapiens_core_60_37e`.`gene` AS A;
CREATE INDEX I_7 ON `homo_sapiens_core_60_37e`.`TEMP7`(`gene_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP8` AS SELECT A.*,B.`alt_allele_id` AS `alt_allele_id_101` FROM `homo_sapiens_core_60_37e`.`TEMP7` AS A INNER JOIN `homo_sapiens_core_60_37e`.`alt_allele` AS B ON A.`gene_id_1021`=B.`gene_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP7`;
CREATE INDEX I_8 ON `homo_sapiens_core_60_37e`.`TEMP8`(`gene_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP9` AS SELECT A.`gene_id_1021`,B.`alt_allele_id_101` FROM `homo_sapiens_core_60_37e`.`gene` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`TEMP8` AS B ON A.`gene_id_1021`=B.`gene_id_1021`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP8`;
RENAME TABLE `homo_sapiens_core_60_37e`.`TEMP9` TO `homo_sapiens_core_60_37e`.`gene__alt_allele`;
CREATE INDEX I_9 ON `homo_sapiens_core_60_37e`.`gene__alt_allele`(`gene_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP10` AS SELECT A.`gene_id_1021` FROM `homo_sapiens_core_60_37e`.`gene` AS A;
CREATE INDEX I_10 ON `homo_sapiens_core_60_37e`.`TEMP10`(`gene_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP11` AS SELECT A.*,B.`value` AS `value_1023`,B.`attrib_type_id` AS `attrib_type_id_1023` FROM `homo_sapiens_core_60_37e`.`TEMP10` AS A INNER JOIN `homo_sapiens_core_60_37e`.`gene_attrib` AS B ON A.`gene_id_1021`=B.`gene_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP10`;
CREATE INDEX I_11 ON `homo_sapiens_core_60_37e`.`TEMP11`(`attrib_type_id_1023`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP12` AS SELECT A.*,B.`description` AS `description_106`,B.`name` AS `name_106`,B.`code` AS `code_106` FROM `homo_sapiens_core_60_37e`.`TEMP11` AS A INNER JOIN `homo_sapiens_core_60_37e`.`attrib_type` AS B ON A.`attrib_type_id_1023`=B.`attrib_type_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP11`;
CREATE INDEX I_12 ON `homo_sapiens_core_60_37e`.`TEMP12`(`gene_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP13` AS SELECT A.`gene_id_1021`,B.`attrib_type_id_1023`,B.`code_106`,B.`description_106`,B.`name_106`,B.`value_1023` FROM `homo_sapiens_core_60_37e`.`gene` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`TEMP12` AS B ON A.`gene_id_1021`=B.`gene_id_1021`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP12`;
RENAME TABLE `homo_sapiens_core_60_37e`.`TEMP13` TO `homo_sapiens_core_60_37e`.`gene__gene_attrib`;
CREATE INDEX I_13 ON `homo_sapiens_core_60_37e`.`gene__gene_attrib`(`gene_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP14` AS SELECT A.`gene_id_1021` FROM `homo_sapiens_core_60_37e`.`gene` AS A;
CREATE INDEX I_14 ON `homo_sapiens_core_60_37e`.`TEMP14`(`gene_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP15` AS SELECT A.*,B.`seq_region_id` AS `seq_region_id_1057`,B.`seq_region_start` AS `seq_region_start_1057`,B.`name` AS `name_1057`,B.`splicing_event_id` AS `splicing_event_id_1057`,B.`seq_region_end` AS `seq_region_end_1057`,B.`seq_region_strand` AS `seq_region_strand_1057`,B.`attrib_type_id` AS `attrib_type_id_1057` FROM `homo_sapiens_core_60_37e`.`TEMP14` AS A INNER JOIN `homo_sapiens_core_60_37e`.`splicing_event` AS B ON A.`gene_id_1021`=B.`gene_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP14`;
CREATE INDEX I_15 ON `homo_sapiens_core_60_37e`.`TEMP15`(`attrib_type_id_1057`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP16` AS SELECT A.*,B.`description` AS `description_106`,B.`name` AS `name_106`,B.`code` AS `code_106` FROM `homo_sapiens_core_60_37e`.`TEMP15` AS A INNER JOIN `homo_sapiens_core_60_37e`.`attrib_type` AS B ON A.`attrib_type_id_1057`=B.`attrib_type_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP15`;
CREATE INDEX I_16 ON `homo_sapiens_core_60_37e`.`TEMP16`(`seq_region_id_1057`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP17` AS SELECT A.*,B.`name` AS `name_1053`,B.`length` AS `length_1053`,B.`coord_system_id` AS `coord_system_id_1053` FROM `homo_sapiens_core_60_37e`.`TEMP16` AS A INNER JOIN `homo_sapiens_core_60_37e`.`seq_region` AS B ON A.`seq_region_id_1057`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP16`;
CREATE INDEX I_17 ON `homo_sapiens_core_60_37e`.`TEMP17`(`coord_system_id_1053`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP18` AS SELECT A.*,B.`rank` AS `rank_107`,B.`name` AS `name_107`,B.`attrib` AS `attrib_107`,B.`species_id` AS `species_id_107`,B.`version` AS `version_107` FROM `homo_sapiens_core_60_37e`.`TEMP17` AS A INNER JOIN `homo_sapiens_core_60_37e`.`coord_system` AS B ON A.`coord_system_id_1053`=B.`coord_system_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP17`;
CREATE INDEX I_18 ON `homo_sapiens_core_60_37e`.`TEMP18`(`seq_region_id_1057`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP19` AS SELECT A.*,B.`sequence` AS `sequence_1013` FROM `homo_sapiens_core_60_37e`.`TEMP18` AS A INNER JOIN `homo_sapiens_core_60_37e`.`dna` AS B ON A.`seq_region_id_1057`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP18`;
CREATE INDEX I_19 ON `homo_sapiens_core_60_37e`.`TEMP19`(`seq_region_id_1057`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP20` AS SELECT A.*,B.`sequence` AS `sequence_1015`,B.`n_line` AS `n_line_1015` FROM `homo_sapiens_core_60_37e`.`TEMP19` AS A INNER JOIN `homo_sapiens_core_60_37e`.`dnac` AS B ON A.`seq_region_id_1057`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP19`;
CREATE INDEX I_20 ON `homo_sapiens_core_60_37e`.`TEMP20`(`gene_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP21` AS SELECT A.`gene_id_1021`,B.`attrib_107`,B.`attrib_type_id_1057`,B.`code_106`,B.`coord_system_id_1053`,B.`description_106`,B.`length_1053`,B.`n_line_1015`,B.`name_1053`,B.`name_1057`,B.`name_106`,B.`name_107`,B.`rank_107`,B.`seq_region_end_1057`,B.`seq_region_id_1057`,B.`seq_region_start_1057`,B.`seq_region_strand_1057`,B.`sequence_1013`,B.`sequence_1015`,B.`species_id_107`,B.`splicing_event_id_1057`,B.`version_107` FROM `homo_sapiens_core_60_37e`.`gene` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`TEMP20` AS B ON A.`gene_id_1021`=B.`gene_id_1021`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP20`;
RENAME TABLE `homo_sapiens_core_60_37e`.`TEMP21` TO `homo_sapiens_core_60_37e`.`gene__splicing_event`;
CREATE INDEX I_21 ON `homo_sapiens_core_60_37e`.`gene__splicing_event`(`gene_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP22` AS SELECT A.`gene_id_1021` FROM `homo_sapiens_core_60_37e`.`gene` AS A;
CREATE INDEX I_22 ON `homo_sapiens_core_60_37e`.`TEMP22`(`gene_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP23` AS SELECT A.*,B.`interaction_type` AS `interaction_type_1069`,B.`transcript_id` AS `transcript_id_1069` FROM `homo_sapiens_core_60_37e`.`TEMP22` AS A INNER JOIN `homo_sapiens_core_60_37e`.`unconventional_transcript_association` AS B ON A.`gene_id_1021`=B.`gene_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP22`;
CREATE INDEX I_23 ON `homo_sapiens_core_60_37e`.`TEMP23`(`gene_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP24` AS SELECT A.`gene_id_1021`,B.`interaction_type_1069`,B.`transcript_id_1069` FROM `homo_sapiens_core_60_37e`.`gene` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`TEMP23` AS B ON A.`gene_id_1021`=B.`gene_id_1021`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP23`;
RENAME TABLE `homo_sapiens_core_60_37e`.`TEMP24` TO `homo_sapiens_core_60_37e`.`gene__unconventional_transcript_association`;
CREATE INDEX I_24 ON `homo_sapiens_core_60_37e`.`gene__unconventional_transcript_association`(`gene_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP25` AS SELECT A.`module_version_102`,A.`db_102`,A.`program_version_102`,A.`program_102`,A.`description_1021`,A.`canonical_transcript_id_1021`,A.`modified_date_1024`,A.`length_1053`,A.`module_102`,A.`name_107`,A.`biotype_1021`,A.`seq_region_id_1021`,A.`program_file_102`,A.`gene_id_1021`,A.`logic_name_102`,A.`db_file_102`,A.`sequence_1015`,A.`version_1024`,A.`is_current_1021`,A.`sequence_1013`,A.`db_version_102`,A.`status_1021`,A.`seq_region_start_1021`,A.`source_1021`,A.`gff_feature_102`,A.`attrib_107`,A.`version_107`,A.`seq_region_strand_1021`,A.`parameters_102`,A.`stable_id_1024`,A.`gff_source_102`,A.`n_line_1015`,A.`seq_region_end_1021`,A.`rank_107`,A.`species_id_107`,A.`display_xref_id_1021`,A.`created_date_1024`,A.`name_1053`,A.`analysis_id_1021`,A.`coord_system_id_1053`,A.`canonical_annotation_1021`,A.`created_102` FROM `homo_sapiens_core_60_37e`.`gene` AS A;
CREATE INDEX I_25 ON `homo_sapiens_core_60_37e`.`TEMP25`(`gene_id_1021`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP26` AS SELECT A.*,B.`seq_region_id` AS `seq_region_id_1062`,B.`transcript_id` AS `transcript_id_1062`,B.`status` AS `status_1062`,B.`seq_region_start` AS `seq_region_start_1062`,B.`description` AS `description_1062`,B.`analysis_id` AS `analysis_id_1062`,B.`biotype` AS `biotype_1062`,B.`display_xref_id` AS `display_xref_id_1062`,B.`is_current` AS `is_current_1062`,B.`seq_region_end` AS `seq_region_end_1062`,B.`seq_region_strand` AS `seq_region_strand_1062`,B.`canonical_translation_id` AS `canonical_translation_id_1062` FROM `homo_sapiens_core_60_37e`.`TEMP25` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`transcript` AS B ON A.`gene_id_1021`=B.`gene_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP25`;
CREATE INDEX I_26 ON `homo_sapiens_core_60_37e`.`TEMP26`(`analysis_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP27` AS SELECT A.*,B.`db` AS `db_102_r1`,B.`db_version` AS `db_version_102_r1`,B.`module` AS `module_102_r1`,B.`gff_source` AS `gff_source_102_r1`,B.`module_version` AS `module_version_102_r1`,B.`logic_name` AS `logic_name_102_r1`,B.`gff_feature` AS `gff_feature_102_r1`,B.`program_version` AS `program_version_102_r1`,B.`db_file` AS `db_file_102_r1`,B.`program_file` AS `program_file_102_r1`,B.`created` AS `created_102_r1`,B.`program` AS `program_102_r1`,B.`parameters` AS `parameters_102_r1` FROM `homo_sapiens_core_60_37e`.`TEMP26` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`analysis` AS B ON A.`analysis_id_1062`=B.`analysis_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP26`;
CREATE INDEX I_27 ON `homo_sapiens_core_60_37e`.`TEMP27`(`seq_region_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP28` AS SELECT A.*,B.`name` AS `name_1053_r1`,B.`length` AS `length_1053_r1`,B.`coord_system_id` AS `coord_system_id_1053_r1` FROM `homo_sapiens_core_60_37e`.`TEMP27` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`seq_region` AS B ON A.`seq_region_id_1062`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP27`;
CREATE INDEX I_28 ON `homo_sapiens_core_60_37e`.`TEMP28`(`coord_system_id_1053_r1`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP29` AS SELECT A.*,B.`rank` AS `rank_107_r1`,B.`name` AS `name_107_r1`,B.`attrib` AS `attrib_107_r1`,B.`species_id` AS `species_id_107_r1`,B.`version` AS `version_107_r1` FROM `homo_sapiens_core_60_37e`.`TEMP28` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`coord_system` AS B ON A.`coord_system_id_1053_r1`=B.`coord_system_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP28`;
CREATE INDEX I_29 ON `homo_sapiens_core_60_37e`.`TEMP29`(`seq_region_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP30` AS SELECT A.*,B.`sequence` AS `sequence_1013_r1` FROM `homo_sapiens_core_60_37e`.`TEMP29` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`dna` AS B ON A.`seq_region_id_1062`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP29`;
CREATE INDEX I_30 ON `homo_sapiens_core_60_37e`.`TEMP30`(`seq_region_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP31` AS SELECT A.*,B.`sequence` AS `sequence_1015_r1`,B.`n_line` AS `n_line_1015_r1` FROM `homo_sapiens_core_60_37e`.`TEMP30` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`dnac` AS B ON A.`seq_region_id_1062`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP30`;
CREATE INDEX I_31 ON `homo_sapiens_core_60_37e`.`TEMP31`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP32` AS SELECT A.*,B.`created_date` AS `created_date_1064`,B.`stable_id` AS `stable_id_1064`,B.`modified_date` AS `modified_date_1064`,B.`version` AS `version_1064` FROM `homo_sapiens_core_60_37e`.`TEMP31` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`transcript_stable_id` AS B ON A.`transcript_id_1062`=B.`transcript_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP31`;
RENAME TABLE `homo_sapiens_core_60_37e`.`TEMP32` TO `homo_sapiens_core_60_37e`.`gene__transcript`;
CREATE INDEX I_32 ON `homo_sapiens_core_60_37e`.`gene__transcript`(`gene_id_1021`);
CREATE INDEX I_33 ON `homo_sapiens_core_60_37e`.`gene__transcript`(`transcript_id_1062`);
ALTER TABLE `homo_sapiens_core_60_37e`.`gene` ADD `gene__transcript_count` INTEGER DEFAULT 0;
UPDATE `homo_sapiens_core_60_37e`.`gene` A SET `gene__transcript_count`=(SELECT COUNT(1) FROM `homo_sapiens_core_60_37e`.`gene__transcript` B WHERE A.`gene_id_1021`=B.`gene_id_1021` AND NOT(B.`analysis_id_1021` IS NULL AND B.`analysis_id_1062` IS NULL AND B.`attrib_107` IS NULL AND B.`attrib_107_r1` IS NULL AND B.`attrib_107_r1` IS NULL AND B.`biotype_1021` IS NULL AND B.`biotype_1062` IS NULL AND B.`canonical_annotation_1021` IS NULL AND B.`canonical_transcript_id_1021` IS NULL AND B.`canonical_translation_id_1062` IS NULL AND B.`coord_system_id_1053` IS NULL AND B.`coord_system_id_1053_r1` IS NULL AND B.`coord_system_id_1053_r1` IS NULL AND B.`created_102` IS NULL AND B.`created_102_r1` IS NULL AND B.`created_102_r1` IS NULL AND B.`created_date_1024` IS NULL AND B.`created_date_1064` IS NULL AND B.`db_102` IS NULL AND B.`db_102_r1` IS NULL AND B.`db_102_r1` IS NULL AND B.`db_file_102` IS NULL AND B.`db_file_102_r1` IS NULL AND B.`db_file_102_r1` IS NULL AND B.`db_version_102` IS NULL AND B.`db_version_102_r1` IS NULL AND B.`db_version_102_r1` IS NULL AND B.`description_1021` IS NULL AND B.`description_1062` IS NULL AND B.`display_xref_id_1021` IS NULL AND B.`display_xref_id_1062` IS NULL AND B.`gff_feature_102` IS NULL AND B.`gff_feature_102_r1` IS NULL AND B.`gff_feature_102_r1` IS NULL AND B.`gff_source_102` IS NULL AND B.`gff_source_102_r1` IS NULL AND B.`gff_source_102_r1` IS NULL AND B.`is_current_1021` IS NULL AND B.`is_current_1062` IS NULL AND B.`length_1053` IS NULL AND B.`length_1053_r1` IS NULL AND B.`length_1053_r1` IS NULL AND B.`logic_name_102` IS NULL AND B.`logic_name_102_r1` IS NULL AND B.`logic_name_102_r1` IS NULL AND B.`modified_date_1024` IS NULL AND B.`modified_date_1064` IS NULL AND B.`module_102` IS NULL AND B.`module_102_r1` IS NULL AND B.`module_102_r1` IS NULL AND B.`module_version_102` IS NULL AND B.`module_version_102_r1` IS NULL AND B.`module_version_102_r1` IS NULL AND B.`n_line_1015` IS NULL AND B.`n_line_1015_r1` IS NULL AND B.`n_line_1015_r1` IS NULL AND B.`name_1053` IS NULL AND B.`name_1053_r1` IS NULL AND B.`name_1053_r1` IS NULL AND B.`name_107` IS NULL AND B.`name_107_r1` IS NULL AND B.`name_107_r1` IS NULL AND B.`parameters_102` IS NULL AND B.`parameters_102_r1` IS NULL AND B.`parameters_102_r1` IS NULL AND B.`program_102` IS NULL AND B.`program_102_r1` IS NULL AND B.`program_102_r1` IS NULL AND B.`program_file_102` IS NULL AND B.`program_file_102_r1` IS NULL AND B.`program_file_102_r1` IS NULL AND B.`program_version_102` IS NULL AND B.`program_version_102_r1` IS NULL AND B.`program_version_102_r1` IS NULL AND B.`rank_107` IS NULL AND B.`rank_107_r1` IS NULL AND B.`rank_107_r1` IS NULL AND B.`seq_region_end_1021` IS NULL AND B.`seq_region_end_1062` IS NULL AND B.`seq_region_id_1021` IS NULL AND B.`seq_region_id_1062` IS NULL AND B.`seq_region_start_1021` IS NULL AND B.`seq_region_start_1062` IS NULL AND B.`seq_region_strand_1021` IS NULL AND B.`seq_region_strand_1062` IS NULL AND B.`sequence_1013` IS NULL AND B.`sequence_1013_r1` IS NULL AND B.`sequence_1013_r1` IS NULL AND B.`sequence_1015` IS NULL AND B.`sequence_1015_r1` IS NULL AND B.`sequence_1015_r1` IS NULL AND B.`source_1021` IS NULL AND B.`species_id_107` IS NULL AND B.`species_id_107_r1` IS NULL AND B.`species_id_107_r1` IS NULL AND B.`stable_id_1024` IS NULL AND B.`stable_id_1064` IS NULL AND B.`status_1021` IS NULL AND B.`status_1062` IS NULL AND B.`transcript_id_1062` IS NULL AND B.`version_1024` IS NULL AND B.`version_1064` IS NULL AND B.`version_107` IS NULL AND B.`version_107_r1` IS NULL AND B.`version_107_r1` IS NULL));
CREATE INDEX I_34 ON `homo_sapiens_core_60_37e`.`gene`(`gene__transcript_count`);
ALTER TABLE `homo_sapiens_core_60_37e`.`gene__transcript` ADD `gene__transcript_count` INTEGER DEFAULT 0;
UPDATE `homo_sapiens_core_60_37e`.`gene__transcript` A SET `gene__transcript_count`=(SELECT MAX(`gene__transcript_count`) FROM `homo_sapiens_core_60_37e`.`gene` B WHERE A.`gene_id_1021`=B.`gene_id_1021`);
CREATE INDEX I_35 ON `homo_sapiens_core_60_37e`.`gene__transcript`(`gene__transcript_count`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP33` AS SELECT A.`transcript_id_1062` FROM `homo_sapiens_core_60_37e`.`gene__transcript` AS A;
CREATE INDEX I_36 ON `homo_sapiens_core_60_37e`.`TEMP33`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP34` AS SELECT A.*,B.`rank` AS `rank_1018`,B.`exon_id` AS `exon_id_1018` FROM `homo_sapiens_core_60_37e`.`TEMP33` AS A INNER JOIN `homo_sapiens_core_60_37e`.`exon_transcript` AS B ON A.`transcript_id_1062`=B.`transcript_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP33`;
CREATE INDEX I_37 ON `homo_sapiens_core_60_37e`.`TEMP34`(`exon_id_1018`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP35` AS SELECT A.*,B.`seq_region_id` AS `seq_region_id_1016`,B.`seq_region_start` AS `seq_region_start_1016`,B.`is_current` AS `is_current_1016`,B.`end_phase` AS `end_phase_1016`,B.`phase` AS `phase_1016`,B.`seq_region_end` AS `seq_region_end_1016`,B.`seq_region_strand` AS `seq_region_strand_1016`,B.`is_constitutive` AS `is_constitutive_1016` FROM `homo_sapiens_core_60_37e`.`TEMP34` AS A INNER JOIN `homo_sapiens_core_60_37e`.`exon` AS B ON A.`exon_id_1018`=B.`exon_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP34`;
CREATE INDEX I_38 ON `homo_sapiens_core_60_37e`.`TEMP35`(`exon_id_1018`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP36` AS SELECT A.*,B.`created_date` AS `created_date_1017`,B.`stable_id` AS `stable_id_1017`,B.`modified_date` AS `modified_date_1017`,B.`version` AS `version_1017` FROM `homo_sapiens_core_60_37e`.`TEMP35` AS A INNER JOIN `homo_sapiens_core_60_37e`.`exon_stable_id` AS B ON A.`exon_id_1018`=B.`exon_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP35`;
CREATE INDEX I_39 ON `homo_sapiens_core_60_37e`.`TEMP36`(`seq_region_id_1016`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP37` AS SELECT A.*,B.`name` AS `name_1053`,B.`length` AS `length_1053`,B.`coord_system_id` AS `coord_system_id_1053` FROM `homo_sapiens_core_60_37e`.`TEMP36` AS A INNER JOIN `homo_sapiens_core_60_37e`.`seq_region` AS B ON A.`seq_region_id_1016`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP36`;
CREATE INDEX I_40 ON `homo_sapiens_core_60_37e`.`TEMP37`(`coord_system_id_1053`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP38` AS SELECT A.*,B.`rank` AS `rank_107`,B.`name` AS `name_107`,B.`attrib` AS `attrib_107`,B.`species_id` AS `species_id_107`,B.`version` AS `version_107` FROM `homo_sapiens_core_60_37e`.`TEMP37` AS A INNER JOIN `homo_sapiens_core_60_37e`.`coord_system` AS B ON A.`coord_system_id_1053`=B.`coord_system_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP37`;
CREATE INDEX I_41 ON `homo_sapiens_core_60_37e`.`TEMP38`(`seq_region_id_1016`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP39` AS SELECT A.*,B.`sequence` AS `sequence_1013` FROM `homo_sapiens_core_60_37e`.`TEMP38` AS A INNER JOIN `homo_sapiens_core_60_37e`.`dna` AS B ON A.`seq_region_id_1016`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP38`;
CREATE INDEX I_42 ON `homo_sapiens_core_60_37e`.`TEMP39`(`seq_region_id_1016`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP40` AS SELECT A.*,B.`sequence` AS `sequence_1015`,B.`n_line` AS `n_line_1015` FROM `homo_sapiens_core_60_37e`.`TEMP39` AS A INNER JOIN `homo_sapiens_core_60_37e`.`dnac` AS B ON A.`seq_region_id_1016`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP39`;
CREATE INDEX I_43 ON `homo_sapiens_core_60_37e`.`TEMP40`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP41` AS SELECT A.`transcript_id_1062`,B.`attrib_107`,B.`coord_system_id_1053`,B.`created_date_1017`,B.`end_phase_1016`,B.`exon_id_1018`,B.`is_constitutive_1016`,B.`is_current_1016`,B.`length_1053`,B.`modified_date_1017`,B.`n_line_1015`,B.`name_1053`,B.`name_107`,B.`phase_1016`,B.`rank_1018`,B.`rank_107`,B.`seq_region_end_1016`,B.`seq_region_id_1016`,B.`seq_region_start_1016`,B.`seq_region_strand_1016`,B.`sequence_1013`,B.`sequence_1015`,B.`species_id_107`,B.`stable_id_1017`,B.`version_1017`,B.`version_107` FROM `homo_sapiens_core_60_37e`.`gene__transcript` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`TEMP40` AS B ON A.`transcript_id_1062`=B.`transcript_id_1062`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP40`;
RENAME TABLE `homo_sapiens_core_60_37e`.`TEMP41` TO `homo_sapiens_core_60_37e`.`transcript__exon_transcript`;
CREATE INDEX I_44 ON `homo_sapiens_core_60_37e`.`transcript__exon_transcript`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP42` AS SELECT A.`transcript_id_1062` FROM `homo_sapiens_core_60_37e`.`gene__transcript` AS A;
CREATE INDEX I_45 ON `homo_sapiens_core_60_37e`.`TEMP42`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP43` AS SELECT A.*,B.`splicing_event_feature_id` AS `splicing_event_feature_id_1058`,B.`start` AS `start_1058`,B.`feature_order` AS `feature_order_1058`,B.`exon_id` AS `exon_id_1058`,B.`transcript_association` AS `transcript_association_1058`,B.`splicing_event_id` AS `splicing_event_id_1058`,B.`type` AS `type_1058`,B.`end` AS `end_1058` FROM `homo_sapiens_core_60_37e`.`TEMP42` AS A INNER JOIN `homo_sapiens_core_60_37e`.`splicing_event_feature` AS B ON A.`transcript_id_1062`=B.`transcript_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP42`;
CREATE INDEX I_46 ON `homo_sapiens_core_60_37e`.`TEMP43`(`exon_id_1058`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP44` AS SELECT A.*,B.`seq_region_id` AS `seq_region_id_1016`,B.`seq_region_start` AS `seq_region_start_1016`,B.`is_current` AS `is_current_1016`,B.`end_phase` AS `end_phase_1016`,B.`phase` AS `phase_1016`,B.`seq_region_end` AS `seq_region_end_1016`,B.`seq_region_strand` AS `seq_region_strand_1016`,B.`is_constitutive` AS `is_constitutive_1016` FROM `homo_sapiens_core_60_37e`.`TEMP43` AS A INNER JOIN `homo_sapiens_core_60_37e`.`exon` AS B ON A.`exon_id_1058`=B.`exon_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP43`;
CREATE INDEX I_47 ON `homo_sapiens_core_60_37e`.`TEMP44`(`exon_id_1058`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP45` AS SELECT A.*,B.`created_date` AS `created_date_1017`,B.`stable_id` AS `stable_id_1017`,B.`modified_date` AS `modified_date_1017`,B.`version` AS `version_1017` FROM `homo_sapiens_core_60_37e`.`TEMP44` AS A INNER JOIN `homo_sapiens_core_60_37e`.`exon_stable_id` AS B ON A.`exon_id_1058`=B.`exon_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP44`;
CREATE INDEX I_48 ON `homo_sapiens_core_60_37e`.`TEMP45`(`seq_region_id_1016`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP46` AS SELECT A.*,B.`name` AS `name_1053`,B.`length` AS `length_1053`,B.`coord_system_id` AS `coord_system_id_1053` FROM `homo_sapiens_core_60_37e`.`TEMP45` AS A INNER JOIN `homo_sapiens_core_60_37e`.`seq_region` AS B ON A.`seq_region_id_1016`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP45`;
CREATE INDEX I_49 ON `homo_sapiens_core_60_37e`.`TEMP46`(`coord_system_id_1053`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP47` AS SELECT A.*,B.`rank` AS `rank_107`,B.`name` AS `name_107`,B.`attrib` AS `attrib_107`,B.`species_id` AS `species_id_107`,B.`version` AS `version_107` FROM `homo_sapiens_core_60_37e`.`TEMP46` AS A INNER JOIN `homo_sapiens_core_60_37e`.`coord_system` AS B ON A.`coord_system_id_1053`=B.`coord_system_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP46`;
CREATE INDEX I_50 ON `homo_sapiens_core_60_37e`.`TEMP47`(`seq_region_id_1016`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP48` AS SELECT A.*,B.`sequence` AS `sequence_1013` FROM `homo_sapiens_core_60_37e`.`TEMP47` AS A INNER JOIN `homo_sapiens_core_60_37e`.`dna` AS B ON A.`seq_region_id_1016`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP47`;
CREATE INDEX I_51 ON `homo_sapiens_core_60_37e`.`TEMP48`(`seq_region_id_1016`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP49` AS SELECT A.*,B.`sequence` AS `sequence_1015`,B.`n_line` AS `n_line_1015` FROM `homo_sapiens_core_60_37e`.`TEMP48` AS A INNER JOIN `homo_sapiens_core_60_37e`.`dnac` AS B ON A.`seq_region_id_1016`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP48`;
CREATE INDEX I_52 ON `homo_sapiens_core_60_37e`.`TEMP49`(`splicing_event_id_1058`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP50` AS SELECT A.*,B.`seq_region_id` AS `seq_region_id_1057`,B.`seq_region_start` AS `seq_region_start_1057`,B.`name` AS `name_1057`,B.`gene_id` AS `gene_id_1057`,B.`seq_region_end` AS `seq_region_end_1057`,B.`seq_region_strand` AS `seq_region_strand_1057`,B.`attrib_type_id` AS `attrib_type_id_1057` FROM `homo_sapiens_core_60_37e`.`TEMP49` AS A INNER JOIN `homo_sapiens_core_60_37e`.`splicing_event` AS B ON A.`splicing_event_id_1058`=B.`splicing_event_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP49`;
CREATE INDEX I_53 ON `homo_sapiens_core_60_37e`.`TEMP50`(`attrib_type_id_1057`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP51` AS SELECT A.*,B.`description` AS `description_106`,B.`name` AS `name_106`,B.`code` AS `code_106` FROM `homo_sapiens_core_60_37e`.`TEMP50` AS A INNER JOIN `homo_sapiens_core_60_37e`.`attrib_type` AS B ON A.`attrib_type_id_1057`=B.`attrib_type_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP50`;
CREATE INDEX I_54 ON `homo_sapiens_core_60_37e`.`TEMP51`(`seq_region_id_1057`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP52` AS SELECT A.*,B.`name` AS `name_1153`,B.`length` AS `length_1153`,B.`coord_system_id` AS `coord_system_id_1153` FROM `homo_sapiens_core_60_37e`.`TEMP51` AS A INNER JOIN `homo_sapiens_core_60_37e`.`seq_region` AS B ON A.`seq_region_id_1057`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP51`;
CREATE INDEX I_55 ON `homo_sapiens_core_60_37e`.`TEMP52`(`coord_system_id_1153`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP53` AS SELECT A.*,B.`rank` AS `rank_117`,B.`name` AS `name_117`,B.`attrib` AS `attrib_117`,B.`species_id` AS `species_id_117`,B.`version` AS `version_117` FROM `homo_sapiens_core_60_37e`.`TEMP52` AS A INNER JOIN `homo_sapiens_core_60_37e`.`coord_system` AS B ON A.`coord_system_id_1153`=B.`coord_system_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP52`;
CREATE INDEX I_56 ON `homo_sapiens_core_60_37e`.`TEMP53`(`seq_region_id_1057`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP54` AS SELECT A.*,B.`sequence` AS `sequence_1113` FROM `homo_sapiens_core_60_37e`.`TEMP53` AS A INNER JOIN `homo_sapiens_core_60_37e`.`dna` AS B ON A.`seq_region_id_1057`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP53`;
CREATE INDEX I_57 ON `homo_sapiens_core_60_37e`.`TEMP54`(`seq_region_id_1057`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP55` AS SELECT A.*,B.`sequence` AS `sequence_1115`,B.`n_line` AS `n_line_1115` FROM `homo_sapiens_core_60_37e`.`TEMP54` AS A INNER JOIN `homo_sapiens_core_60_37e`.`dnac` AS B ON A.`seq_region_id_1057`=B.`seq_region_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP54`;
CREATE INDEX I_58 ON `homo_sapiens_core_60_37e`.`TEMP55`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP56` AS SELECT A.`transcript_id_1062`,B.`attrib_107`,B.`attrib_117`,B.`attrib_type_id_1057`,B.`code_106`,B.`coord_system_id_1053`,B.`coord_system_id_1153`,B.`created_date_1017`,B.`description_106`,B.`end_1058`,B.`end_phase_1016`,B.`exon_id_1058`,B.`feature_order_1058`,B.`gene_id_1057`,B.`is_constitutive_1016`,B.`is_current_1016`,B.`length_1053`,B.`length_1153`,B.`modified_date_1017`,B.`n_line_1015`,B.`n_line_1115`,B.`name_1053`,B.`name_1057`,B.`name_106`,B.`name_107`,B.`name_1153`,B.`name_117`,B.`phase_1016`,B.`rank_107`,B.`rank_117`,B.`seq_region_end_1016`,B.`seq_region_end_1057`,B.`seq_region_id_1016`,B.`seq_region_id_1057`,B.`seq_region_start_1016`,B.`seq_region_start_1057`,B.`seq_region_strand_1016`,B.`seq_region_strand_1057`,B.`sequence_1013`,B.`sequence_1015`,B.`sequence_1113`,B.`sequence_1115`,B.`species_id_107`,B.`species_id_117`,B.`splicing_event_feature_id_1058`,B.`splicing_event_id_1058`,B.`stable_id_1017`,B.`start_1058`,B.`transcript_association_1058`,B.`type_1058`,B.`version_1017`,B.`version_107`,B.`version_117` FROM `homo_sapiens_core_60_37e`.`gene__transcript` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`TEMP55` AS B ON A.`transcript_id_1062`=B.`transcript_id_1062`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP55`;
RENAME TABLE `homo_sapiens_core_60_37e`.`TEMP56` TO `homo_sapiens_core_60_37e`.`transcript__splicing_event_feature`;
CREATE INDEX I_59 ON `homo_sapiens_core_60_37e`.`transcript__splicing_event_feature`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP57` AS SELECT A.`transcript_id_1062` FROM `homo_sapiens_core_60_37e`.`gene__transcript` AS A;
CREATE INDEX I_60 ON `homo_sapiens_core_60_37e`.`TEMP57`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP58` AS SELECT A.*,B.`value` AS `value_1063`,B.`attrib_type_id` AS `attrib_type_id_1063` FROM `homo_sapiens_core_60_37e`.`TEMP57` AS A INNER JOIN `homo_sapiens_core_60_37e`.`transcript_attrib` AS B ON A.`transcript_id_1062`=B.`transcript_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP57`;
CREATE INDEX I_61 ON `homo_sapiens_core_60_37e`.`TEMP58`(`attrib_type_id_1063`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP59` AS SELECT A.*,B.`description` AS `description_106`,B.`name` AS `name_106`,B.`code` AS `code_106` FROM `homo_sapiens_core_60_37e`.`TEMP58` AS A INNER JOIN `homo_sapiens_core_60_37e`.`attrib_type` AS B ON A.`attrib_type_id_1063`=B.`attrib_type_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP58`;
CREATE INDEX I_62 ON `homo_sapiens_core_60_37e`.`TEMP59`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP60` AS SELECT A.`transcript_id_1062`,B.`attrib_type_id_1063`,B.`code_106`,B.`description_106`,B.`name_106`,B.`value_1063` FROM `homo_sapiens_core_60_37e`.`gene__transcript` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`TEMP59` AS B ON A.`transcript_id_1062`=B.`transcript_id_1062`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP59`;
RENAME TABLE `homo_sapiens_core_60_37e`.`TEMP60` TO `homo_sapiens_core_60_37e`.`transcript__transcript_attrib`;
CREATE INDEX I_63 ON `homo_sapiens_core_60_37e`.`transcript__transcript_attrib`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP61` AS SELECT A.`transcript_id_1062` FROM `homo_sapiens_core_60_37e`.`gene__transcript` AS A;
CREATE INDEX I_64 ON `homo_sapiens_core_60_37e`.`TEMP61`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP62` AS SELECT A.*,B.`feature_type` AS `feature_type_1065`,B.`feature_id` AS `feature_id_1065` FROM `homo_sapiens_core_60_37e`.`TEMP61` AS A INNER JOIN `homo_sapiens_core_60_37e`.`transcript_supporting_feature` AS B ON A.`transcript_id_1062`=B.`transcript_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP61`;
CREATE INDEX I_65 ON `homo_sapiens_core_60_37e`.`TEMP62`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP63` AS SELECT A.`transcript_id_1062`,B.`feature_id_1065`,B.`feature_type_1065` FROM `homo_sapiens_core_60_37e`.`gene__transcript` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`TEMP62` AS B ON A.`transcript_id_1062`=B.`transcript_id_1062`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP62`;
RENAME TABLE `homo_sapiens_core_60_37e`.`TEMP63` TO `homo_sapiens_core_60_37e`.`transcript__transcript_supporting_feature`;
CREATE INDEX I_66 ON `homo_sapiens_core_60_37e`.`transcript__transcript_supporting_feature`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP64` AS SELECT A.`transcript_id_1062` FROM `homo_sapiens_core_60_37e`.`gene__transcript` AS A;
CREATE INDEX I_67 ON `homo_sapiens_core_60_37e`.`TEMP64`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP65` AS SELECT A.*,B.`interaction_type` AS `interaction_type_1069`,B.`gene_id` AS `gene_id_1069` FROM `homo_sapiens_core_60_37e`.`TEMP64` AS A INNER JOIN `homo_sapiens_core_60_37e`.`unconventional_transcript_association` AS B ON A.`transcript_id_1062`=B.`transcript_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP64`;
CREATE INDEX I_68 ON `homo_sapiens_core_60_37e`.`TEMP65`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP66` AS SELECT A.`transcript_id_1062`,B.`gene_id_1069`,B.`interaction_type_1069` FROM `homo_sapiens_core_60_37e`.`gene__transcript` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`TEMP65` AS B ON A.`transcript_id_1062`=B.`transcript_id_1062`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP65`;
RENAME TABLE `homo_sapiens_core_60_37e`.`TEMP66` TO `homo_sapiens_core_60_37e`.`transcript__unconventional_transcript_association`;
CREATE INDEX I_69 ON `homo_sapiens_core_60_37e`.`transcript__unconventional_transcript_association`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP67` AS SELECT A.`transcript_id_1062`,A.`module_version_102`,A.`species_id_107_r1`,A.`program_version_102`,A.`description_1021`,A.`is_current_1062`,A.`module_version_102_r1`,A.`canonical_transcript_id_1021`,A.`seq_region_id_1062`,A.`program_version_102_r1`,A.`length_1053`,A.`name_107`,A.`sequence_1015_r1`,A.`coord_system_id_1053_r1`,A.`version_107_r1`,A.`seq_region_id_1021`,A.`db_file_102_r1`,A.`gene_id_1021`,A.`name_1053_r1`,A.`logic_name_102`,A.`parameters_102_r1`,A.`db_102_r1`,A.`version_1024`,A.`is_current_1021`,A.`sequence_1013`,A.`analysis_id_1062`,A.`status_1021`,A.`attrib_107_r1`,A.`seq_region_start_1021`,A.`source_1021`,A.`gff_feature_102`,A.`n_line_1015_r1`,A.`attrib_107`,A.`version_107`,A.`seq_region_end_1062`,A.`parameters_102`,A.`stable_id_1024`,A.`gff_source_102`,A.`name_107_r1`,A.`seq_region_end_1021`,A.`display_xref_id_1021`,A.`name_1053`,A.`analysis_id_1021`,A.`status_1062`,A.`seq_region_start_1062`,A.`coord_system_id_1053`,A.`canonical_annotation_1021`,A.`created_102`,A.`display_xref_id_1062`,A.`gene__transcript_count`,A.`program_102_r1`,A.`db_102`,A.`canonical_translation_id_1062`,A.`program_102`,A.`gff_feature_102_r1`,A.`sequence_1013_r1`,A.`created_date_1064`,A.`modified_date_1024`,A.`modified_date_1064`,A.`module_102`,A.`biotype_1021`,A.`program_file_102`,A.`sequence_1015`,A.`db_file_102`,A.`db_version_102_r1`,A.`biotype_1062`,A.`stable_id_1064`,A.`rank_107_r1`,A.`db_version_102`,A.`seq_region_strand_1062`,A.`gff_source_102_r1`,A.`version_1064`,A.`created_102_r1`,A.`logic_name_102_r1`,A.`seq_region_strand_1021`,A.`module_102_r1`,A.`n_line_1015`,A.`rank_107`,A.`species_id_107`,A.`program_file_102_r1`,A.`created_date_1024`,A.`length_1053_r1`,A.`description_1062` FROM `homo_sapiens_core_60_37e`.`gene__transcript` AS A;
CREATE INDEX I_70 ON `homo_sapiens_core_60_37e`.`TEMP67`(`transcript_id_1062`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP68` AS SELECT A.*,B.`seq_start` AS `seq_start_1066`,B.`start_exon_id` AS `start_exon_id_1066`,B.`end_exon_id` AS `end_exon_id_1066`,B.`seq_end` AS `seq_end_1066`,B.`translation_id` AS `translation_id_1066` FROM `homo_sapiens_core_60_37e`.`TEMP67` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`translation` AS B ON A.`transcript_id_1062`=B.`transcript_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP67`;
CREATE INDEX I_71 ON `homo_sapiens_core_60_37e`.`TEMP68`(`translation_id_1066`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP69` AS SELECT A.*,B.`created_date` AS `created_date_1068`,B.`stable_id` AS `stable_id_1068`,B.`modified_date` AS `modified_date_1068`,B.`version` AS `version_1068` FROM `homo_sapiens_core_60_37e`.`TEMP68` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`translation_stable_id` AS B ON A.`translation_id_1066`=B.`translation_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP68`;
RENAME TABLE `homo_sapiens_core_60_37e`.`TEMP69` TO `homo_sapiens_core_60_37e`.`transcript__translation`;
CREATE INDEX I_72 ON `homo_sapiens_core_60_37e`.`transcript__translation`(`transcript_id_1062`);
CREATE INDEX I_73 ON `homo_sapiens_core_60_37e`.`transcript__translation`(`translation_id_1066`);
ALTER TABLE `homo_sapiens_core_60_37e`.`gene__transcript` ADD `transcript__translation_count` INTEGER DEFAULT 0;
UPDATE `homo_sapiens_core_60_37e`.`gene__transcript` A SET `transcript__translation_count`=(SELECT COUNT(1) FROM `homo_sapiens_core_60_37e`.`transcript__translation` B WHERE A.`transcript_id_1062`=B.`transcript_id_1062` AND NOT(B.`analysis_id_1021` IS NULL AND B.`analysis_id_1062` IS NULL AND B.`attrib_107` IS NULL AND B.`attrib_107_r1` IS NULL AND B.`biotype_1021` IS NULL AND B.`biotype_1062` IS NULL AND B.`canonical_annotation_1021` IS NULL AND B.`canonical_transcript_id_1021` IS NULL AND B.`canonical_translation_id_1062` IS NULL AND B.`coord_system_id_1053` IS NULL AND B.`coord_system_id_1053_r1` IS NULL AND B.`created_102` IS NULL AND B.`created_102_r1` IS NULL AND B.`created_date_1024` IS NULL AND B.`created_date_1064` IS NULL AND B.`created_date_1068` IS NULL AND B.`db_102` IS NULL AND B.`db_102_r1` IS NULL AND B.`db_file_102` IS NULL AND B.`db_file_102_r1` IS NULL AND B.`db_version_102` IS NULL AND B.`db_version_102_r1` IS NULL AND B.`description_1021` IS NULL AND B.`description_1062` IS NULL AND B.`display_xref_id_1021` IS NULL AND B.`display_xref_id_1062` IS NULL AND B.`end_exon_id_1066` IS NULL AND B.`gene_id_1021` IS NULL AND B.`gff_feature_102` IS NULL AND B.`gff_feature_102_r1` IS NULL AND B.`gff_source_102` IS NULL AND B.`gff_source_102_r1` IS NULL AND B.`is_current_1021` IS NULL AND B.`is_current_1062` IS NULL AND B.`length_1053` IS NULL AND B.`length_1053_r1` IS NULL AND B.`logic_name_102` IS NULL AND B.`logic_name_102_r1` IS NULL AND B.`modified_date_1024` IS NULL AND B.`modified_date_1064` IS NULL AND B.`modified_date_1068` IS NULL AND B.`module_102` IS NULL AND B.`module_102_r1` IS NULL AND B.`module_version_102` IS NULL AND B.`module_version_102_r1` IS NULL AND B.`n_line_1015` IS NULL AND B.`n_line_1015_r1` IS NULL AND B.`name_1053` IS NULL AND B.`name_1053_r1` IS NULL AND B.`name_107` IS NULL AND B.`name_107_r1` IS NULL AND B.`parameters_102` IS NULL AND B.`parameters_102_r1` IS NULL AND B.`program_102` IS NULL AND B.`program_102_r1` IS NULL AND B.`program_file_102` IS NULL AND B.`program_file_102_r1` IS NULL AND B.`program_version_102` IS NULL AND B.`program_version_102_r1` IS NULL AND B.`rank_107` IS NULL AND B.`rank_107_r1` IS NULL AND B.`seq_end_1066` IS NULL AND B.`seq_region_end_1021` IS NULL AND B.`seq_region_end_1062` IS NULL AND B.`seq_region_id_1021` IS NULL AND B.`seq_region_id_1062` IS NULL AND B.`seq_region_start_1021` IS NULL AND B.`seq_region_start_1062` IS NULL AND B.`seq_region_strand_1021` IS NULL AND B.`seq_region_strand_1062` IS NULL AND B.`seq_start_1066` IS NULL AND B.`sequence_1013` IS NULL AND B.`sequence_1013_r1` IS NULL AND B.`sequence_1015` IS NULL AND B.`sequence_1015_r1` IS NULL AND B.`source_1021` IS NULL AND B.`species_id_107` IS NULL AND B.`species_id_107_r1` IS NULL AND B.`stable_id_1024` IS NULL AND B.`stable_id_1064` IS NULL AND B.`stable_id_1068` IS NULL AND B.`start_exon_id_1066` IS NULL AND B.`status_1021` IS NULL AND B.`status_1062` IS NULL AND B.`translation_id_1066` IS NULL AND B.`version_1024` IS NULL AND B.`version_1064` IS NULL AND B.`version_1068` IS NULL AND B.`version_107` IS NULL AND B.`version_107_r1` IS NULL));
CREATE INDEX I_74 ON `homo_sapiens_core_60_37e`.`gene__transcript`(`transcript__translation_count`);
ALTER TABLE `homo_sapiens_core_60_37e`.`transcript__translation` ADD `transcript__translation_count` INTEGER DEFAULT 0;
UPDATE `homo_sapiens_core_60_37e`.`transcript__translation` A SET `transcript__translation_count`=(SELECT MAX(`transcript__translation_count`) FROM `homo_sapiens_core_60_37e`.`gene__transcript` B WHERE A.`transcript_id_1062`=B.`transcript_id_1062`);
CREATE INDEX I_75 ON `homo_sapiens_core_60_37e`.`transcript__translation`(`transcript__translation_count`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP70` AS SELECT A.`translation_id_1066` FROM `homo_sapiens_core_60_37e`.`transcript__translation` AS A;
CREATE INDEX I_76 ON `homo_sapiens_core_60_37e`.`TEMP70`(`translation_id_1066`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP71` AS SELECT A.*,B.`seq_start` AS `seq_start_1047`,B.`hit_end` AS `hit_end_1047`,B.`perc_ident` AS `perc_ident_1047`,B.`analysis_id` AS `analysis_id_1047`,B.`hit_start` AS `hit_start_1047`,B.`hit_name` AS `hit_name_1047`,B.`score` AS `score_1047`,B.`protein_feature_id` AS `protein_feature_id_1047`,B.`seq_end` AS `seq_end_1047`,B.`evalue` AS `evalue_1047`,B.`external_data` AS `external_data_1047` FROM `homo_sapiens_core_60_37e`.`TEMP70` AS A INNER JOIN `homo_sapiens_core_60_37e`.`protein_feature` AS B ON A.`translation_id_1066`=B.`translation_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP70`;
CREATE INDEX I_77 ON `homo_sapiens_core_60_37e`.`TEMP71`(`analysis_id_1047`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP72` AS SELECT A.*,B.`db` AS `db_102`,B.`db_version` AS `db_version_102`,B.`module` AS `module_102`,B.`gff_source` AS `gff_source_102`,B.`module_version` AS `module_version_102`,B.`logic_name` AS `logic_name_102`,B.`gff_feature` AS `gff_feature_102`,B.`program_version` AS `program_version_102`,B.`db_file` AS `db_file_102`,B.`program_file` AS `program_file_102`,B.`created` AS `created_102`,B.`program` AS `program_102`,B.`parameters` AS `parameters_102` FROM `homo_sapiens_core_60_37e`.`TEMP71` AS A INNER JOIN `homo_sapiens_core_60_37e`.`analysis` AS B ON A.`analysis_id_1047`=B.`analysis_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP71`;
CREATE INDEX I_78 ON `homo_sapiens_core_60_37e`.`TEMP72`(`translation_id_1066`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP73` AS SELECT A.`translation_id_1066`,B.`analysis_id_1047`,B.`created_102`,B.`db_102`,B.`db_file_102`,B.`db_version_102`,B.`evalue_1047`,B.`external_data_1047`,B.`gff_feature_102`,B.`gff_source_102`,B.`hit_end_1047`,B.`hit_name_1047`,B.`hit_start_1047`,B.`logic_name_102`,B.`module_102`,B.`module_version_102`,B.`parameters_102`,B.`perc_ident_1047`,B.`program_102`,B.`program_file_102`,B.`program_version_102`,B.`protein_feature_id_1047`,B.`score_1047`,B.`seq_end_1047`,B.`seq_start_1047` FROM `homo_sapiens_core_60_37e`.`transcript__translation` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`TEMP72` AS B ON A.`translation_id_1066`=B.`translation_id_1066`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP72`;
RENAME TABLE `homo_sapiens_core_60_37e`.`TEMP73` TO `homo_sapiens_core_60_37e`.`translation__protein_feature`;
CREATE INDEX I_79 ON `homo_sapiens_core_60_37e`.`translation__protein_feature`(`translation_id_1066`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP74` AS SELECT A.`translation_id_1066` FROM `homo_sapiens_core_60_37e`.`transcript__translation` AS A;
CREATE INDEX I_80 ON `homo_sapiens_core_60_37e`.`TEMP74`(`translation_id_1066`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP75` AS SELECT A.*,B.`value` AS `value_1067`,B.`attrib_type_id` AS `attrib_type_id_1067` FROM `homo_sapiens_core_60_37e`.`TEMP74` AS A INNER JOIN `homo_sapiens_core_60_37e`.`translation_attrib` AS B ON A.`translation_id_1066`=B.`translation_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP74`;
CREATE INDEX I_81 ON `homo_sapiens_core_60_37e`.`TEMP75`(`attrib_type_id_1067`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP76` AS SELECT A.*,B.`description` AS `description_106`,B.`name` AS `name_106`,B.`code` AS `code_106` FROM `homo_sapiens_core_60_37e`.`TEMP75` AS A INNER JOIN `homo_sapiens_core_60_37e`.`attrib_type` AS B ON A.`attrib_type_id_1067`=B.`attrib_type_id`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP75`;
CREATE INDEX I_82 ON `homo_sapiens_core_60_37e`.`TEMP76`(`translation_id_1066`);
CREATE TABLE `homo_sapiens_core_60_37e`.`TEMP77` AS SELECT A.`translation_id_1066`,B.`attrib_type_id_1067`,B.`code_106`,B.`description_106`,B.`name_106`,B.`value_1067` FROM `homo_sapiens_core_60_37e`.`transcript__translation` AS A LEFT JOIN `homo_sapiens_core_60_37e`.`TEMP76` AS B ON A.`translation_id_1066`=B.`translation_id_1066`;
DROP TABLE `homo_sapiens_core_60_37e`.`TEMP76`;
RENAME TABLE `homo_sapiens_core_60_37e`.`TEMP77` TO `homo_sapiens_core_60_37e`.`translation__translation_attrib`;
CREATE INDEX I_83 ON `homo_sapiens_core_60_37e`.`translation__translation_attrib`(`translation_id_1066`);
